gax-httpjson/src/test/java/com/google/api/gax/httpjson/HttpHeadersUtilsTest.java

```diff
@@ -0,0 +1,133 @@
+package com.google.api.gax.httpjson;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import com.google.api.client.http.HttpHeaders;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.junit.Test;
+
+/*
```

Comment:
The license needs to be before the package / import lines.

Reply:
Oops..